storage/gcsemu/filestore.go

```diff
@@ -51,11 +70,25 @@ func (fs *filestore) GetBucketMeta(baseUrl HttpBaseUrl, bucket string) (*storage
 	}
 
 	obj := BucketMeta(baseUrl, bucket)
+	fMeta := metaFilename(f)
+	buf, err := ioutil.ReadFile(fMeta)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return nil, fmt.Errorf("could not read metadata file %s: %w", fMeta, err)
+		}
+	}
+
+	if len(buf) != 0 {
+		if err := json.NewDecoder(bytes.NewReader(buf)).Decode(obj); err != nil {
+			return nil, fmt.Errorf("could not parse file attributes %q for %s: %w", buf, f, err)
+		}
+	}
+
```

Comment:
So this change is just to allow Buckets to hold metadata?  Is it specifically for `TimeCreated`, or is the idea that you can get/set more sophisticated bucket metadata?

Reply:
for now it is for TimeCreated but may be used for ther metadata as well if needed in the future

Reply:
What about a less-invasive change to just supply TimeCreated, as you've done in the memstore?